### PR TITLE
SSCS-8496 Process reasonable adjustment event callback

### PR DESCRIPTION
**Pull request by jackmaloney**
https://tools.hmcts.net/jira/browse/SSCS-8496

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processreasonableadjustment/ProcessReasonableAdjustmentAboutToSubmitHandler.java
Comment (ryandac) [inline src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/processreasonableadjustment/ProcessReasonableAdjustmentAboutToSubmitHandler.java:40]: Can call sscsCaseData.updateReasonableAdjustmentsOutstanding() instead

File: build.gradle
Comment (ryandac) [inline build.gradle:310]: Can use version 4.1.35.RC-SSCS-8495-1

Comment (ryandac) [reply build.gradle:310]: should also keep sscs-pdf-email-common to 1.4.20-RC-SSCS-8495-1

**Review by ryandac — CHANGES_REQUESTED** [review]

File: build.gradle
Comment (jackmaloney) [reply build.gradle:310]: Yeh that was intentional as wanted to wait until 8495 was merged. I've now rebased my branch and created a new  4.1.35.RC-SSCS-8495-3 version

**Review by ryandac — APPROVED** [review]

**Pull request merged**
